src/GeneralStateTestsFiller/EIPTests/stEOF/stEIP3540/EOF1_CallsFiller.yml

```diff
@@ -498,7 +413,7 @@ EOF1_Calls:
     - indexes:
        data: ':label eof_staticcall_eof'
       network:
-        - 'Merge+3540+3670'
+        - 'Shanghai'
       result:
         a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
           nonce: 1
```

Comment:
Would be nice to have a little comment for the storage values below, what they intent to indicate.

Reply:
Where exacty would you like this? If we add a a comment in every expectation, it will be very repetitive. It's basically everywhere:
- slot 0 - result of CALL/CREATE/CREATE2
- slot 1 - always 1 when not OOG